ci/scripts/e2e-risedev.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+echo "--- Adjust permission"
+chmod +x ./target/debug/risingwave
+chmod +x ./target/debug/risedev-playground
+
+echo "--- Generate RiseDev CI config"
+cp risedev-components.ci.env risedev-components.user.env
+
+echo "--- Prepare RiseDev playground"
+~/cargo-make/makers pre-start-playground
+~/cargo-make/makers link-all-in-one-binaries
+
+echo "--- e2e, ci-3cn-1fe, streaming"
```

Comment:
Is lines beginning with `---` regarded as a step separator in buildkite? If not, can we separate them into different steps like GitHub Action?